### DISCUSSION TITLE
archives/3ecc

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#11] [General]
# archives/3ecc

网易代理MC的三大原罪——资本与社区的矛盾  引言 不管是在国际版玩的人还是在网易版玩的人，对于网易的代理基本都没有一个正向的评价。最常看到的抱怨是“网易外挂太多” “网易限制太大” “网易只想圈钱”一类的浮于表面，最直接的体现网易代理弊端的问题。但是我认为这些表面的问题并不能很好的说明为什么网易代理如此遭人唾弃的根本原因。试问：“如果网易真的解决了这些表面的问题就可以得到一个正向的评价吗？这些

https://5l.re/archives/3ecc

**Comment by YiLingwu2023** [comment]
點讚
